src/orion/benchmark/__init__.py

```diff
@@ -21,7 +21,13 @@ class Benchmark:
     name: str
         Name of the benchmark
     algorithms: list, optional
-        Algorithms used for benchmark, each algorithm can be a string or dict.
+        Algorithms used for benchmark, each algorithm definition will be a dict with two keys.
+
+        algorithm: str or dict
```

Comment:
Is `str` still supported? That would be preferable.

Reply:
oh, so yes, but we need to pass [{'algorithm': 'algo'}] all the time. That seems very cumbersome. :worried:

Reply:
Yes, supporting str as well would be better.